api/xds_protocol.rst

```diff
@@ -469,3 +469,155 @@ occurred via a :ref:`resource update <xds_protocol_resource_update>`.
+to a subset of routes that are likely useful to the Envoy instance. For
+example, these might be filtered down by routes that exist in the current
+namespace. The xDS management server will send this same collection of
+resources on reconnect.
+
+.. figure:: diagrams/delta_rds_connection.svg
+   :alt: Delta RDS connection
+
+Virtual Host Resource Naming Convention
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Virtual hosts in VHDS are identified by a combination of the name of the route
+configuration to which the virtual host belong as well as the host HTTP "host"
+header (authority for HTTP2) entry. Resources should be named as follows:
+
+<route configuration name>/<host entry>
+
+Note that matching should be done from right to left since a host entry cannot
+contain slashes while a route configuration name can. 
+
+Requesting Additional resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Envoy will send a
+:ref:`DeltaDiscoveryRequest <envoy_api_msg_DeltaDiscoveryRequest>` with the
+:ref:`resource_names_subscribe <envoy_api_field_DeltaDiscoveryRequest.resource_names_subscribe>`
+field populated with the virtual host resource names of each of the resources
+that it would like to subscribe to. Each of the virtual hosts contained in the
+:ref:`DeltaDiscoveryRequest's <envoy_api_msg_DeltaDiscoveryRequest>`
+resources field will be added to the route configuration maintained by Envoy.
+If Envoy's route configuration already contains a given virtual host, it will
+be overwritten by data received in the
+:ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>`
+but only if the updated virtual host is different from its current state.
+The :ref:`resource.aliases <envoy_api_field_resource.aliases>` field contains
+all host/authority header values used to create the on-demand request. The
+configuration server needs to keep track of virtual hosts known to Envoy.
+
+If a virtual host is requested for which the management server does not know
+about, then the management server should respond with a 
+:ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>` in which
+the :ref:`resources <envoy_api_field_DeltaDiscoveryResponse.resources>` entry
+for that virtual host has the :ref:`name <envoy_api_field_resource.name>` and
+:ref:`aliases <envoy_api_field_resource.aliases>` set to the requested host
+entry and the resource unpopulated. This will allow Envoy to match the
+requested resource to the response.
+
+Envoy should respond to the 
+:ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>`
+with an Ack/Nack 
+:ref:`DeltaDiscoveryRequest's <envoy_api_msg_DeltaDiscoveryRequest>`.
+
+.. figure:: diagrams/delta_rds_request_additional_resources.svg
+   :alt: Delta RDS request additional resources
+
+Request Path Resolution
++++++++++++++++++++++++
+If a route for the contents of the host/authority header cannot be resolved:
+- A :ref:`DeltaDiscoveryRequest <envoy_api_msg_DeltaDiscoveryRequest>` as 
```

Comment:
Is this going to format correctly?  Github's rendering doesn't seem to show this as a bulletted list, although I'm not sure what RST will actually do.

Reply:
Fixed this. Needed a new line before the bullet.